cwms-data-api/src/test/java/cwms/cda/helpers/DTOMatch.java

```diff
@@ -45,8 +53,10 @@ private DTOMatch() {
 
     public static void assertMatch(CwmsId first, CwmsId second, String variableName) {
         assertAll(
-            () -> Assertions.assertEquals(first.getOfficeId(), second.getOfficeId(), variableName + " is not the same. Office ID differs"),
```

Comment:
Honestly, I would just add an @SuppressWarning for the line length to these methods or the class, even though they're long they read better on one line.

Reply:
ok, revised